grpc-gateway/pb/cloudConfiguration.proto

```diff
@@ -11,12 +11,14 @@ message CloudConfigurationResponse {
     // provides a current time of server in nanoseconds.
     int64 current_time = 1;
     string device_onboarding_code_url  = 2;
-    string jwt_claim_owner_id = 3;
+    string jwt_owner_claim = 3;
+    string jwt_device_id_claim = 11;
     string signing_server_address  = 4;
     string cloud_id = 5;
     string cloud_url  = 6;
     string cloud_authorization_provider = 7;
     string cloud_certificate_authorities = 8;
+    string device_authority = 10;
```

Comment:
There is nothing like device authority. There is only one server issuing tokens, and this one is called authority.
Please rename to `authorization_server`
Please update documentation of authority everywhere to:
```
Authority is the address of the token-issuing authentication server. Services will use this URI to find and retrieve the public key that can be used to validate the token’s signature.
```

Reply:
fixed